README.md

```diff
@@ -1,12 +1,13 @@
 # git-machete
 
-[![Join the chat at https://gitter.im/VirtusLab/git-machete](https://badges.gitter.im/VirtusLab/git-machete.svg)](https://gitter.im/VirtusLab/git-machete)
```

Comment:
Gitter's kinda dead anyway (at least gitter for this project, for sure)... the emergence of Github Discussions didn't help it for sure in general (and for most projects including this one, issues are a more natural place for conversation)